src/main/java/org/jabref/gui/util/DynamicallyChangeableNode.java

```diff
@@ -0,0 +1,16 @@
+package org.jabref.gui.util;
+
+import javafx.scene.Node;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.VBox;
+
+/*
+ * A node that can change it's content using a setContent(Node node) method, similar to {@link Tab}.
+ */
```

Comment:
```suggestion
/**
 * A node that can change its content using a setContent(Node node) method, similar to {@link Tab}.
 */
```

Reply:
Mabe, you can add some reasoning why this is introduced. Maybe, because it makes the code more readable and VBox does not directly support content replacement.